packages/buidler-core/test/internal/buidler-evm/stack-traces/test-files/0_5/special-cases/function-type-as-public-parameter/c.sol

```diff
@@ -0,0 +1,14 @@
+pragma solidity ^0.5.0;
+
+import "./../../../../../../../../console.sol";
+
+contract C {
+
+  function printSomething() public {
+    console.log("something");
+  }
+
+  function proxyMessageCall(function (uint256) external returns (uint256) aFunction, uint256 aParameter) public returns (uint256) {
```

Comment:
You can receive functions like that? :o How do you call this thing?

Reply:
This is actually supported since v0.4.5 according to the changelog... I recall seeing it briefly some time ago but never found a way to use it. You can see some hints about it in the documentation [here](https://solidity.readthedocs.io/en/v0.7.1/types.html#function-types).